src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -21,9 +21,21 @@
  */
 class InputOption
 {
+    /**
+     * Do not accept input for this option (e.g. --yell). This is the default behavior of options;
+     */
     public const VALUE_NONE = 1;
+    /**
+     * This value is required (e.g. --iterations=5 or -i5), the option itself is still optional;
```

Comment:
```suggestion
     * A value must be passed when the option is used (e.g. --iterations=5 or -i5)
```

An option is always optional per se, I think we can omit that part.

Reply:
agree. maybe it should also be changed in the offical docs then?

Reply:
I think it's fine if the docs has additional statements to remind the obvious :)